client/app/components/proptypes.js

```diff
@@ -65,7 +65,7 @@ export const Query = PropTypes.shape({
 
 export const AlertOptions = PropTypes.shape({
   column: PropTypes.string,
-  selector: PropTypes.oneOf(["first", "min", "max"]),
+  selector: PropTypes.oneOf(["first", "min", "max", "average"]),
```

Comment:
let's use "avg" for consistency (or change the other options to "minimum", "maximum", ...)